src/components/header/OverflowEdgeIndicator.js

```diff
@@ -2,21 +2,40 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
 import React, { PureComponent, PropTypes } from 'react';
 import classNames from 'classnames';
 
 import './OverflowEdgeIndicator.css';
 
+type Props = {
+  className: string,
+  children: number,
+};
+
 class OverflowEdgeIndicator extends PureComponent {
-  constructor(props) {
+  props: Props;
+  state: {
+    overflowsOnTop: ?boolean,
```

Comment:
actually, looks like that updating the flow-typed definition for `classnames` makes it work. They replaced `?boolean` by `*`, see https://github.com/flowtype/flow-typed/blob/master/definitions/npm/classnames_v2.x.x/flow_v0.25.x-/classnames_v2.x.x.js#L3

This was changed in https://github.com/flowtype/flow-typed/commit/3c3f096590c6149599c4fac65bb8d841d009e8dc